graphql_compiler/schema_transformation/utils.py

```diff
@@ -33,7 +34,33 @@ class InvalidTypeNameError(SchemaTransformError):
 
 
 class SchemaNameConflictError(SchemaTransformError):
-    """Raised when renaming types or fields cause name conflicts."""
+    """Raised when renaming or merging types or fields cause name conflicts."""
+
+
+_alphanumeric_and_underscore = frozenset(string.ascii_letters + string.digits + '_')
+
+
+def check_schema_identifier_is_valid(identifier):
```

Comment:
Similar function [here](https://github.com/kensho-technologies/graphql-compiler/blob/master/graphql_compiler/compiler/helpers.py#L147)

Granted, that function lets through other things (see line 150 of that file in particular) and it's not documented as well as this one is, but could be worth a shot modifying it to reuse that code?

Reply:
It's probably not worth the effort -- they check for very different identifiers, and the logic is quite simple. The necessary top-level utils file is probably going to be more trouble than it's worth.